examples/devenv-django/devenv.nix

```diff
@@ -0,0 +1,87 @@
+{ pkgs, ... }:
+
+let
+  db_user = "postgres";
+  db_host = "localhost";
+  db_port = "5432";
+  db_name = "db";
+  python_version = "3.10";
+in
+{
+  packages = [ pkgs.git pkgs.postgresql_14 ];
+
+  languages.python = {
+    enable = true;
+    poetry.enable = true;
+  };
+
+  env = {
+    PYTHON_VERSION = python_version;
```

Comment:
Should this be part of the python module? (Always be added for python projects?)

Reply:
To simplify this:

```suggestion
    PYTHON_VERSION = config.languages.python.package.version;
```

Reply:
I'm just going to remove `PYTHON_VERSION` here as I was using it to explicitly force `poetry` to use a specific version of `python` as `pkgs.poetry` & `pkgs.python3` are different versions in the repo from which I pulled this example.

Otherwise I'm not sure what exactly the impact of setting this env var is. 

It makes sense to me that we should be explicit in declaring the `python` version via `languages.python.package`